applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -198,31 +198,43 @@ def MarkKuttaElements(self):
             # Marking the elements under the trailing edge as kutta
             if(distance_to_wake < 0.0):
                 elem.SetValue(CPFApp.KUTTA, True)
+    @staticmethod
+    def CheckIfElemIsCutByWake(elem):
+        npos=0
+        nneg=0
+        distances = elem.GetValue(KratosMultiphysics.ELEMENTAL_DISTANCES)
+        for nodal_distance in distances:
+            if nodal_distance>0:
+                npos += 1
```

Comment:
I think we do no need npos since we do not use it.

Reply:
You are right!